src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double height;
+    private final double width;
+
+    public Rectangle(String color, double length, double width) {
+        super(color);
+        this.height = length;
+        this.width = width;
```

Comment:
Don't pass expressions directly in the constructor call. Create separate variables for 'color', 'height', and 'width' and pass them to the super constructor and field assignments for better readability.